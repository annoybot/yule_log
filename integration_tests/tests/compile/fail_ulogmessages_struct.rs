use yule_log::ULogMessages;

#[derive(ULogMessages)]
pub struct NotAnEnum {
    value: u64,
}

fn main() {}
