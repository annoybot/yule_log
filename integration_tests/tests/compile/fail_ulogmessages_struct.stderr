error: ULogMessages derive only works on enums.
 --> tests/compile/fail_ulogmessages_struct.rs:4:12
  |
4 | pub struct NotAnEnum {
  |            ^^^^^^^^^
